mods.conf

```diff
@@ -36,3 +36,4 @@ valhelsia-core                           416935       6296775      # 2025-03-12T
 smartbrainlib                            661293       6668303      # 2025-06-18T21:21:13.86Z   | SmartBrainLib-neoforge-1.21.1-1.16.8.jar
 kotlin-for-forge                         351264       6497906      # 2025-05-05T07:11:02.707Z  | kotlinforforge-5.8.0-all.jar
 roots                                    246183       6664961      # 2025-06-18T06:52:53.79Z   | roots-4.0.0.7-alpha.jar
+enchanted-witchcraft                     560363       6668864
```

Comment:
`6668864` is the latest file for Fabric.
`6668863` is the Neoforge version, which we need here.

And murphy obviously had to hit you with one of the corner cases. Enchanted Witchcraft has extra dependencies on Modopedia and StateObserver. All in all the PR has to look like this:

```
enchanted-witchcraft                     560363       6668863
modopedia                                1132038      6673794
stateobserver                            701213       6522771
```